components/authz-service/server/server.go

```diff
@@ -122,16 +121,16 @@ func NewGRPCServer(ctx context.Context,
 				tracing.ServerInterceptor(tracing.GlobalTracer()),
 				grpc_logrus.UnaryServerInterceptor(logrusEntry, logrusOpts...),
 				InputValidationInterceptor(),
-				v2PolServer.EngineUpdateInterceptor(),
+				polServer.EngineUpdateInterceptor(),
 			),
 		),
 	)
 
 	// register all services
 	health.RegisterHealthServer(g, health.NewService())
-	api_v2.RegisterPoliciesServer(g, v2PolServer)
-	api_v2.RegisterProjectsServer(g, v2ProjectsServer)
-	api_v2.RegisterAuthorizationServer(g, v2AuthzServer)
+	api_v2.RegisterPoliciesServer(g, polServer)
+	api_v2.RegisterProjectsServer(g, projectsServer)
+	api_v2.RegisterAuthorizationServer(g, authzServer)
 	common.RegisterSubjectPurgeServer(g, subjectPurgeServer)
```

Comment:
I'd second @bcmdarroch's coment about eventually merging the subject purge stuff into `PoliciesServer`

Reply:
I'll do that in a follow up because getting rid of `common` will involve touching the other services.